sormas-ui/src/main/java/de/symeda/sormas/ui/events/EventParticipantsView.java

```diff
@@ -300,6 +301,12 @@ public HorizontalLayout createTopBar() {
 
 		}
 
+		if (UserProvider.getCurrent().hasUserRight(UserRight.CASE_CREATE)) {
```

Comment:
should probably be `EVENT_PARTICIPANT_CREATE`

Reply:
updated